libdnf/hy-query.c

```diff
@@ -439,7 +499,7 @@ filter_release(HyQuery q, struct _Filter *f, Map *m)
 
             pool_split_evr(pool, evr, &e, &v, &r);
 
-            if (cmp_type == HY_GLOB) {
+            if (cmp_type & HY_GLOB) {
                 if (fnmatch(match, r, 0))
```

Comment:
Line above accepts `HY_ICASE` now. But fnmatch always get 0.
Is it what you want? Or this:
`if (fnmatch(match, r, cmp_type & HY_ICASE ? FNM_CASEFOLD : 0))`

Reply:
I don't think this is part of the PR. Only it changes a problem, but don't want to create functionality